diff -uraN shotwell-33.alpha/meson.build omv-shotwell-33.alpha/meson.build
--- shotwell-33.alpha/meson.build	2025-01-27 19:16:56.000000000 +0100
+++ omv-shotwell-33.alpha/meson.build	2026-04-06 21:25:55.793970696 +0200
@@ -87,7 +87,7 @@
 gstreamer_pbu = dependency('gstreamer-pbutils-1.0', version : '>= 1.20')
 gphoto2 = dependency('libgphoto2', version : '>= 2.5.0')
 gudev = dependency('gudev-1.0', version : '>= 145', required: false)
-gexiv2 = dependency('gexiv2', version: '>= 0.12.3')
+gexiv2 = dependency('gexiv2-0.16', version: '>= 0.15.0')
 libraw = dependency('libraw', version : '>= 0.13.2')
 libexif = dependency('libexif', version : '>= 0.6.16')
 secret = dependency('libsecret-1', required: true)
