Index: shotwell-0.10.1/plugins/plugins.mk
===================================================================
--- shotwell-0.10.1.orig/plugins/plugins.mk
+++ shotwell-0.10.1/plugins/plugins.mk
@@ -16,7 +16,8 @@ EXTRA_PLUGINS := \
 EXTRA_PLUGINS_RC := \
 	plugins/shotwell-publishing-extras/yandex_publish_model.glade \
 	plugins/shotwell-publishing-extras/piwigo.png \
-	plugins/shotwell-publishing-extras/piwigo_authentication_pane.glade
+	plugins/shotwell-publishing-extras/piwigo_authentication_pane.glade \
+	plugins/shotwell-publishing-extras/piwigo_publishing_options_pane.glade
 
 ALL_PLUGINS := $(PLUGINS) $(EXTRA_PLUGINS)
 
Index: shotwell-0.10.1/plugins/shotwell-publishing-extras/Makefile
===================================================================
--- shotwell-0.10.1.orig/plugins/shotwell-publishing-extras/Makefile
+++ shotwell-0.10.1/plugins/shotwell-publishing-extras/Makefile
@@ -19,7 +19,8 @@ SRC_FILES := \
 RC_FILES := \
 	yandex_publish_model.glade \
 	piwigo.png \
-	piwigo_authentication_pane.glade
+	piwigo_authentication_pane.glade \
+	piwigo_publishing_options_pane.glade
 
 include ../Makefile.plugin.mk
 
